packages/now-python/src/index.ts

```diff
@@ -116,9 +146,11 @@ export const build = async ({
 
     // Python needs to know where to look up all the packages we just installed.
     // We tell it to use the same location as used with `--target`
+    const pythonPath = process.env.PYTHONPATH;
     process.env.PYTHONPATH = tempDir;
     const convertCmd = join(tempDir, 'bin', 'pipfile2req');
     await pipenvConvert(convertCmd, pipfileLockDir);
+    process.env.PYTHONPATH = pythonPath;
```

Comment:
Why set PYTHONPATH twice? I think the comment above needs to be updated

Reply:
In order to reset `PYTHONPATH` to its original value again. It is only set to find `pipfile2req`, but after `await pipenvConvert(convertCmd, pipfileLockDir);` has been executed, any packges installed in `tempDir` should not be in the search path anymore for the rest of the script.

I just noticed that `process.env.PYTHONPATH = pythonPath;` is actually not correct. It should be:
```ts
if (pythonPath === undefined) {
  delete process.env.PYTHONPATH;
} else {
  process.env.PYTHONPATH = pythonPath;
}
```
I'll fix it once you agree that resetting `process.env.PYTHONPATH` is valid in general.

Reply:
Seems good 👍

Reply:
Done